src/ui/public/styles/dark-theme.less

```diff
@@ -583,6 +583,9 @@
     color: @dark-button-font;
   }
 
+  .truncate-by-height:before {
+    background: linear-gradient(to bottom, transparent 0%, @dashboard-bg 100%) !important;
+  }
```

Comment:
Would you mind tacking on another rule to change the appearance of the _source fields?

```less
  .truncate-by-height:before {
    background: linear-gradient(to bottom, transparent 0%, @dashboard-bg 100%) !important;
  }

  kbn-table, .kbn-table, tbody[kbn-rows] {
    dl.source {
      dt {
        background: #5f5f5f;
        color: #ffffff;
      }
    }
  }
```

This will change the from this:

![image](https://cloud.githubusercontent.com/assets/1238659/22571684/842571cc-e955-11e6-9e18-6f58f0aa1c45.png)

To this:

![image](https://cloud.githubusercontent.com/assets/1238659/22571644/60993b58-e955-11e6-8166-78a2a738478a.png)